core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -465,6 +465,8 @@ object SparkSubmit {
       OptionAssigner(args.archives, YARN, ALL_DEPLOY_MODES, sysProp = "spark.yarn.dist.archives"),
       OptionAssigner(args.principal, YARN, ALL_DEPLOY_MODES, sysProp = "spark.yarn.principal"),
       OptionAssigner(args.keytab, YARN, ALL_DEPLOY_MODES, sysProp = "spark.yarn.keytab"),
+      OptionAssigner(args.callerContext, YARN, ALL_DEPLOY_MODES,
+        sysProp = "spark.hadoop.callerContext"),
```

Comment:
Thanks for this info. I have changed the property name to "spark.upstreamApp.callercontext".